test/.package.json

```diff
@@ -20,5 +25,12 @@
   "dependencies": {
     "prop-types": "^15.5.10",
     "react": "^15.5.4"
+  },
+  "prettier": {
```

Comment:
It's up to the user to provide prettier config. We can create shareable prettier config and then in the main app, we can do something like:

```
// prettier.config.js
module.exports = require('prettier-config-skyscanner');
```